boards/Protectli/VP66xx/results.csv.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2024 3mdeb Sp. z o. o.
+
+SPDX-License-Identifier: CC-BY-SA-4.0
```

Comment:
The results for v66xx have already been published and should not be included here.

Reply:
Yes, this branch needs to be rebased after publishing VP66xx results.